src/defs/server.ts

```diff
@@ -904,24 +949,6 @@ tmpDef.argDescs = {};
 tmpDef.scriptSide = ScriptSide.Server;
 ServerDefinitions.push(tmpDef);
 
-tmpDef = new MTAFunction;
-tmpDef.label = "Vector2";
```

Comment:
Why did you delete the vector and matrix constructors? They're a thing in MTA, just not properly implemented in vscode-mtalua: https://wiki.multitheftauto.com/wiki/Vector/Vector2

Reply:
I didn't delete them I moved them to shared